src/apps/inventory/controllers/book.controller.ts

```diff
@@ -12,6 +20,17 @@ export class BookController {
   @Post('/create')
   @Authorized(UserRole.MANAGER)
   async create(@Body() data: BookCreateDto) {
-    return this.bookService.createGet(data);
+    return await this.bookService.createGet(data);
+  }
+
+  @Get('/detail/:id')
```

Comment:
We should follow the best practice of the Restful API design, therefore the path `@Get('/:id')` is better. 
```
GET /books
GET /books/:id
POST /books
PUT /books/:id
```

Reply:
@quangtran85 updated feedback